content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -206,7 +206,7 @@ point to the CA certificate and key.
 
 ### Managing the kubeadm drop-in file for the kubelet {#kubelet-drop-in}
 
-See [here](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/kubelet-integration/#the-kubelet-drop-in-file-for-systemd) for details on the `kubeadm` integration with the `kubelet`.
+See [here](/docs/setup/production-environment/tools/kubeadm/kubelet-integration/#the-kubelet-drop-in-file-for-systemd) for details on the `kubeadm` integration with the `kubelet`.
```

Comment:
Would you change the `See here` to something more informative. Something like:
See the [setup of the kubelet drop-in file for systemd](/docs/setup/production-environment/tools/kubeadm/kubelet-integration/#the-kubelet-drop-in-file-for-systemd) for details about configuring the `kubelet` with `kubeadm`.

Reply:
@kbhawkey so I had it like that in f128ce7dc2022e424a28ffc3faacb7b9a8af0ad8 because I agree that's better style but then @neolit123 asked for it to be changed to "here" linked in his review 4 days ago above (expand Outdated Show Resolved) so how to play this game here - who's right? :smiley_cat: I also prefer it how I initially had it - @neolit123 will you agree to me reverting what you had asked for?

Reply:
@vorburger 
The docs style guide suggests the preferred way to name hyperlinks:
https://kubernetes.io/docs/contribute/style/style-guide/#links

Reply:
done